src/offline/Restorer.js

```diff
@@ -20,15 +20,22 @@ class Restorer {
    * @return {Promise<ol.Extent>}
    */
   restore(map) {
-    console.log('should restore something');
-    return this.configuration_.getItem('offline_content').then(offlineContent => this.doRestore_(offlineContent));
+    return this.configuration_.getItem('offline_content').then(offlineContent => this.doRestore_(map, offlineContent));
   }
 
   /**
+   * @param {ol.Map} map
    * @param {ngeox.OfflinePersistentContent} offlineContent
    * @return {ol.Extent}
    */
-  doRestore_(offlineContent) {
+  doRestore_(map, offlineContent) {
```

Comment:
this is were we could override what we do with the layers ...

Reply:
add a `@private` or remove the final underscore please

Reply:
OK, done. Anything else to do?